packit/config.py

```diff
@@ -174,8 +176,7 @@ def get_local_package_config(directory=None) -> Optional[PackageConfig]:
             return parse_loaded_config(loaded_config=loaded_config)
 
         logger.debug(f"The local config file '{config_file_name_full}' not found.")
-
-    return None
+    raise RuntimeError("No packit config found.")
```

Comment:
I've created [some exceptions](https://github.com/packit-service/packit/blob/master/packit/exceptions.py) for packit so we can use them.